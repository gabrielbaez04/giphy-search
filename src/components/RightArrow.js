import React from 'react';
import PropTypes from 'prop-types';

const RightArrow = ({ goToNextSlide }) => {
    return (
        <div className="nextArrow arrow" onClick={goToNextSlide}>
            <i className="fa fa-arrow-right fa-2x" aria-hidden="true"></i>
        </div>
    );
}

RightArrow.propTypes = {
    goToNextSlide: PropTypes.func.isRequired
}

export default RightArrow;