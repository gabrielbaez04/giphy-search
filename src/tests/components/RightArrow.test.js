import { shallow } from 'enzyme';
import RightArrow from '../../components/RightArrow';
import React from 'react';

test('should render RightArrow correctly', () => {
    const wrapper = shallow(<RightArrow goToNextSlide={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
});
