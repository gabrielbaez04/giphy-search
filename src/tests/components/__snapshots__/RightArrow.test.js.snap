// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RightArrow correctly 1`] = `
<div
  className="nextArrow arrow"
  onClick={[MockFunction]}
>
  <i
    aria-hidden="true"
    className="fa fa-arrow-right fa-2x"
  />
</div>
`;
